lib/utils.js

```diff
@@ -0,0 +1,12 @@
+import * as _ from 'underscore';
+
+function areObjectsEmpty(a, b) {
```

Comment:
I think this implementation is too specific in that it requires two objects. I think there are already existing methods to check if a single object is empty, and then it's just as easy as doing `if (_.isEmpty(a) && _.isEmpty(b))` and then it makes this utility method go away.

Reply:
`_.isEmpty(8)` is giving me `true`. Because we don't know what Onyx is sending to the connection, we only really want to do this check to avoid unnecessary re-renders on objects (and maybe arrays? I didn't see any array while debugging)

Reply:
There are some arrays, but they are less frequently used.

This one comes to mind: https://github.com/Expensify/App/blob/48437acb34dbc38b9b7a60dae4c272727b8ce3ab/src/libs/actions/ActiveClients.js#L9